litex/soc/integration/export.py

```diff
@@ -308,7 +308,7 @@ def print_svd_register(csr, csr_address, description, length, svd):
         svd.append('                    <addressOffset>0x{:04x}</addressOffset>'.format(csr_address))
         svd.append('                    <resetValue>0x{:02x}</resetValue>'.format(csr.reset_value))
         svd.append('                    <size>{}</size>'.format(length))
-        svd.append('                    <access>{}</access>'.format(csr.access))
+        # svd.append('                    <access>{}</access>'.format(csr.access))  # 'access' is a lie: "read-only" registers can legitimately change state based on a write, and is in fact used to handle the "pending" field in events
```

Comment:
This is a good catch. `read-only` doesn't mean the same thing in LiteX that it does in SVD. Omitting it is probably the best solution here.

What we really want is to be able to indicate that each Field's `modifiedWriteValues` is `oneToClear` within an interrupt pending register, however we don't currently have the ability to do that here.

Reply:
Reference: https://www.keil.com/pack/doc/cmsis/SVD/html/elem_registers.html#elem_field